cmd/gatekeeper/main.go

```diff
@@ -53,6 +54,17 @@ func main() {
 		panic(err)
 	}
 
+	// Check if config file is present in the github repo. If present automatically add the E2E Test Suites check
+	configFileInRepo := gh.FilePresentInRepo(e2eTestConfigFile)
+	if configFileInRepo {
+		fmt.Println("'E2E Test Suites' check automatically added to the required checks")
+		if repoConfig == nil {
+			repoConfig = &config.Repo{RequiredChecks: []string{"E2E Test Suites"}}
+		}else{
+			repoConfig.RequiredChecks = append(repoConfig.RequiredChecks, "E2E Test Suites")
+		}
```

Comment:
No need for the else. Just init the variable if not already set then use the same logic for both cases :) 
```suggestion
		if repoConfig == nil {
			repoConfig = &config.Repo{RequiredChecks: []string{}}
		}
		repoConfig.RequiredChecks = append(repoConfig.RequiredChecks, "E2E Test Suites")
```

Reply:
But thinking about it, maybe we update `GetRepoConfig` to return an empty object if nothing found 🤔  (Maybe an improvement for a separate PR though)